OasisDEX/oasis-borrow
# Adding auto take profit trigger from FE

# [Adding auto take profit trigger from FE](https://app.shortcut.com/oazo-apps/story/5573/ui-auto-take-profit-sending-trigger-to-the-back-end)

  
## Changes 👷‍♀️

- Implemented preparing triggers data from FE
- Handled adding trigger when button is clicked
- Out of scope: middlesteps, progress when trigger is being added etc.
  
## How to test 🧪
Try to add auto take profit trigger here's example transaction https://goerli.etherscan.io/tx/0x24c53bd4a9a7272332e3155f33aa6d45b5b98e1fbd5ad09aca51e51fbeafdaca#eventlog

- Check if FE doesn't crash
- check if trigger is added correctly
- check if data  in trigger is correct
